common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -142,7 +142,8 @@
             log: function(eventName, data) {
                 var logInfo = _.extend({
                     id: this.state.id,
-                    code: this.state.isYoutubeType() ? this.state.youtubeId() : 'html5'
+                    // eslint-disable-next-line no-nested-ternary
+                    code: this.state.isYoutubeType() ? this.state.youtubeId() : this.state.canPlayHLS ? 'hls' : 'html5'
```

Comment:
@mulby or @stroilova This is just heads up ^^ . After this change being merged into master. The code field for event logger could have value "hls" in addition to youtubeID and "html5"